tests/fake_requests.py

```diff
@@ -0,0 +1,12 @@
+import json
+
+
+
+class MockResponse(object):
+    def __init__(self, url=None, headers=None, text=None, status_code=None):
+        self.url = url
+        self.headers = headers
+        self.text = text
+        self.status_code = status_code
+    def json(self):
+        return json.loads(self.text)
```

Comment:
A little bit care about the names.:sweat_smile:
When I want to use `MockResponse`, the import syntax will look like `from tests.fake_requests import MockResponse`. Maybe make it more consistent? For example, `from tests.fake_requests import FakeResponse` or `from tests.mock_requests import MockResponse`.